### PR TITLE
Update tokio, hyper versions

### PR DESCRIPTION
**Pull request by jaspervdm**
Update tokio and hyper

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -115,7 +115,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "0df2f85c8a2abbe3b7d7e748052fdd9b76a0458fdeb16ad4223f5eca78c7c130"
 dependencies = [
  "addr2line",
- "cfg-if",
+ "cfg-if 0.1.10",
```
Comment (antiochp) [inline Cargo.lock:118]: Are all these changes in `Cargo.lock` due to updating tokio and hyper?

File: chain/src/chain.rs [hunk 2]
```diff
@@ -1484,32 +1484,53 @@ impl Chain {
 	/// Migrate our local db from v2 to v3.
 	/// "commit only" inputs.
 	fn migrate_db_v2_v3(store: &ChainStore) -> Result<(), Error> {
+		if store.batch()?.is_blocks_v3_migrated()? {
```
Comment (antiochp) [inline chain/src/chain.rs:1487]: Just merged #3513 so rebasing this PR should clear this lot up.

**Review by antiochp — COMMENTED** [review]
:+1: Looks good to me.
Question about `Cargo.lock` - just to check.

**Review by quentinlesceller — APPROVED** [review]
Need a rebase but otherwise looking good

File: Cargo.lock (hunk 1 above)
Comment (jaspervdm) [reply Cargo.lock:118]: Yes, i did `cargo update -p` for tokio and hyper

File: chain/src/chain.rs (hunk 2 above)
Comment (jaspervdm) [reply chain/src/chain.rs]: Done

**Review by antiochp — APPROVED** [review]

**Comment by antiochp** [comment]
Lets get this merged today and we can think about releasing a `beta.3`.

**Pull request merged**